src/app.js

```diff
@@ -1 +1,22 @@
 'use strict';
+
+const fs = require('fs');
+
+function funcRead() {
```

Comment:
The 'funcRead' function does not take any arguments, which makes it less reusable. Consider parameterizing it, so it could be used in different contexts.